posting/list.go

```diff
@@ -1014,21 +1014,22 @@ func (l *List) Value(readTs uint64) (rval types.Val, rerr error) {
 func (l *List) ValueFor(readTs uint64, langs []string) (rval types.Val, rerr error) {
 	l.RLock() // All public methods should acquire locks, while private ones should assert them.
 	defer l.RUnlock()
-	p, err := l.postingFor(readTs, langs)
+	p, lang, err := l.postingFor(readTs, langs)
```

Comment:
Don't make a lot of changes.

you already passing lang as a parameter. just use it from the input. Anyways you're going to sort based on
one language. Just put an assert in the sort fetch. So you don't break anything unknowingly